sktime/forecasting/adapters/_autots.py

```diff
@@ -0,0 +1,449 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+from __future__ import annotations
+
+__author__ = ["MBristle"]
+
+import logging
+
+import pandas as pd
+
+from sktime.forecasting.base import BaseForecaster, ForecastingHorizon
+
+
+class AutoTSAdapter(BaseForecaster):
+    """Act as adapter for the AutoTS library.
+
+    Parameter:
+    ---------
+        fh (ForecastingHorizon): The forecasting horizon.
```

Comment:
This seems missing from `__init__`, is it correct?

Reply:
Good catch! I used it in that manner but as sktime standard passes the FH in fit or predict, i refactored it but forgot to update the docs.